packages/openmrs/src/Adaptor.js

```diff
@@ -60,23 +36,7 @@ async function login(state) {
  * @returns {Operation}
  */
 export function getPatient(uuid, callback = false) {
-  return state => {
-    Log.info(`Searching for patient with uuid: ${uuid}`);
-    const { instanceUrl } = state.configuration;
-    const defaultQuery = { v: 'full', limit: 1 };
-    const url = `${instanceUrl}/ws/rest/v1/patient/${uuid}`;
-
-    return agent
-      .get(url)
-      .accept('json')
-      .query(defaultQuery)
-      .then(response => {
-        Log.success(`Found patient.`);
```

Comment:
Can you preserve this with `console.log` too?

Reply:
Adding this log requires me to use this method
``` (state)=>sendRequest('GET','patient').(state).then()....```
Using this does not allow me to get the response needed when run this job but the logs before calling sendRequests are working.
@taylordowns2000 @mtuchi  What do you advice?